app/Order.php

```diff
@@ -26,6 +26,30 @@ class Order extends Model
         'status',
     ];
 
+    protected $statusOrder = [
+        1 => 'pendding',
+        2 =>'accepted',
+        3 =>'rejected',
+        5 => 'received',
+    ];
+
+    /**
+     * The accessors to append to the model's array form.
+     *
+     * @var array
+     */
+    protected $appends = ['status_order'];
+
+    /**
+     * Get the order's status.
+     *
+     * @return string
+     */
+    public function getStatusOrderAttribute()
```

Comment:
getStatusAttribute($status) {
return $this->statusOrder[$status];
}

update o apidoc lun em